src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -153,6 +153,7 @@ foam.CLASS({
             console.warn('Missing stack, can not push view');
             return;
           }
+          this.__subContext__.setControllerMode('view');
```

Comment:
Why not foam.u2.ControllerMode.VIEW?

Reply:
This is a holdover from comics v2, this is how it manages mementos, it doesnt actually affect the controllermode